### ISSUE TITLE
Error when launching Amethyst Runtime

### ISSUE THREAD

**Issue by notndricim** [FrederoxDev/Amethyst#126]
I get this error when launching Minecraft with only Amethyst Runtime. Same thing happens when i run it with mods.
```
[AmethystProxy] Using 'AmethystProxy@1.1.0'
[AmethystProxy] McThreadID: 12056, McThreadHandle: 0x16c
[AmethystProxy] Injecting runtime 'AmethystRuntime@1.4.1'
[AmethystRuntime] McThreadID: 12056, McThreadHandle: 0x16c
[AmethystRuntime] Using 'AmethystRuntime@1.4.1'
[AmethystRuntime] Minecraft Version: 1.21.3
[AmethystRuntime] Exception thrown at 0x5aced2f58

Disassembly:
       6ECED2F58 mov rax, [rax]
                 ^^^^^^^^^^^^^^
       6ECED2F5B call [rip+0x4374F]
       6ECED2F61 jmp +0x9D

Error Code: 0xc0000005
Press Numpad0 to close...

```

**Comment by FrederoxDev** [comment]
https://aka.ms/vs/17/release/vc_redist.x64.exe
Check you have this installed? lmk if you're still having issues

**Comment by notndricim** [comment]
that fixed it thanks! Do you know which mods are available for 4.1?

**Comment by FrederoxDev** [comment]
all but zoom i think
